Rationale.md

```diff
@@ -1,5 +1,17 @@
 # Design Rationale
 
+This document describes rationales for WebAssembly's design decisions, acting as
+footnotes to the main design text, keeping the main specification easier to
+read, and making it easier to revisit decisions later without having to plow
+through all the issues and pull requests. This rationale document tries to list
+how decisions were made, and where tradeoffs were made for the sake of language
+ergonomics, portability, performance, security, and Getting Things Done.
+
+WebAssembly was designed incrementally, with multiple implementations being
+pursued concurrently. Design desisions are made with a solid rationale, but as
+implementations move forward and developers start using WebAssembly it may turn
+out to that some decisions were suboptimal.
```

Comment:
This final sentence could be made less Eeyore-esque by saying that we're willing to update the design (before v.1 is finalized) as new stuff comes to light... [man](https://www.youtube.com/watch?v=XKVmpJaMffM).  Perhaps:  "... but as the MVP stablizes, implementations appear, and we get experience from real-world codebases, we can revisit the alternatives listed below, reevaluate the tradeoffs and potentially update the design in AstSemantics.md before the MVP is finalized."